llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -10,6 +10,8 @@
 include "iree-dialects/Dialect/LinalgExt/IR/LinalgExtBase.td"
 include "iree-dialects/Dialect/LinalgExt/IR/LinalgExtInterfaces.td"
 include "mlir/Interfaces/ControlFlowInterfaces.td"
+//include "mlir/Interfaces/AggregatedOpInterface.td"
+include "mlir/Dialect/Linalg/IR/LinalgInterfaces.td"
```

Comment:
For the Bazel build, a dependency on `@llvm-project//mlir:LinalgOpsTdFiles` should be added here: https://github.com/openxla/iree/blob/ddb0d7d3746c85a3bfd9ef7a2ecf51e188a94e49/llvm-external-projects/iree-dialects/BUILD.bazel#L250-L258

(see other possible targets here: https://github.com/llvm/llvm-project/blob/5fc69134a15a436e69c5b1db26f0a91c1deb3acb/utils/bazel/llvm-project-overlay/mlir/BUILD.bazel#L10002-L10220)

I think that will fix the error reported here: https://github.com/openxla/iree/actions/runs/6747112501/job/18342521898?pr=15294#step:4:123

```
ERROR: /work/llvm-external-projects/iree-dialects/BUILD.bazel:197:18: TdGenerate llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.cpp.inc failed: (Exit 1): mlir-tblgen failed: error executing command bazel-out/k8-opt-exec-2B5CBBC6/bin/external/llvm-project/mlir/mlir-tblgen --gen-op-defs llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td -I ... (remaining 17 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox
llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td:14:9: error: Could not find include file 'mlir/Dialect/Linalg/IR/LinalgInterfaces.td'
include "mlir/Dialect/Linalg/IR/LinalgInterfaces.td"
        ^
llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td:14:9: error: Unexpected token at top level
include "mlir/Dialect/Linalg/IR/LinalgInterfaces.td"
        ^
```

A similar change may be useful on the CMake side in https://github.com/openxla/iree/blob/main/llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/CMakeLists.txt, but there are other dependency issues in those files (see https://gist.github.com/ScottTodd/07a934cffff4e0b82dc2bd67691e2cab), so I'm not sure what the best approach is there.